lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -141,7 +168,9 @@ If (Test-Path -Path $path -PathType Leaf) {
 ElseIf ($inherit -eq "") {
     $inherit = "ContainerInherit, ObjectInherit"
 }
- 
+
+$ErrorActionPreference = "Stop" 
```

Comment:
I would feel better if the required cmdlets got the "ErrorAction" parameter, instead of a "blanket" ErrorActionPreference - open to feedback on that one.